Assets/MRTK/Core/Inspectors/PropertyDrawers/TypeReferencePropertyDrawer.cs

```diff
@@ -289,17 +289,22 @@ private static Type[] FindTypesByName(string typeName, SystemTypeAttribute filte
         private static void DisplayDropDown(Rect position, List<Type> types, Type selectedType, TypeGrouping grouping)
         {
             var menu = new GenericMenu();
-            menu.AddItem(new GUIContent("(None)"), selectedType == null, OnSelectedTypeName, null);
```

Comment:
My only hesitation would be around if we ever use this property drawer in an optional scenario, where it's not required to choose a type. No examples of that scenario are coming to mind though...so we might be okay.

Reply:
I did a quick run through of all references of SystemType and didn't see any instances where we have good behavior when the SystemType is null 😅